src/test/scala/com/soundcloud/periskop/client/ExceptionExporterSpec.scala

```diff
@@ -114,7 +131,39 @@ class ExceptionExporterSpec extends Specification with Mockito {
           |            "request_headers": {
           |              "User-Agent": "Foobrowser",
           |              "Accept": "text/html"
+          |            },
+          |            "request_body": "body"
+          |          }
+          |        },
+          |        {
+          |          "error": {
+          |            "class": "com.soundcloud.periskop.client.ExceptionExporterSpec$$FakeException",
+          |            "message": "foo1",
+          |            "stacktrace": [
+          |              "kls.mthd(file:1)",
+          |              "kls.mthd(file:2)"
+          |            ],
+          |            "cause": {
+          |              "class": "com.soundcloud.periskop.client.ExceptionExporterSpec$$FakeException",
+          |              "message": "foo1parent",
+          |              "stacktrace": [
+          |                "kls.mthd(file:1)",
+          |                "kls.mthd(file:2)"
+          |              ],
+          |              "cause": null
           |            }
+          |          },
+          |          "severity": "error",
+          |          "uuid": "c3c24195-27ae-4455-ba5a-7b504a7699a4",
+          |          "timestamp": "2018-01-02T11:22:33.000Z",
+          |          "http_context": {
+          |            "request_method": "POST",
+          |            "request_url": "http://foo.com/boo",
+          |            "request_headers": {
+          |              "User-Agent": "Foobrowser",
+          |              "Accept": "text/html"
+          |            },
+          |            "request_body": null
```

Comment:
in my head im following a pattern here cause i've seen some other nulls in this json body + we check for null in `periskop`. I would usually lean towards not sending an optional value rather than sending a null